include/ultra64/vi.h

```diff
@@ -92,4 +97,6 @@ typedef struct {
 #define OS_VI_PAL_HPN2      26
 #define OS_VI_PAL_HPF2      27
 
+#define OS_VI_UNK28         28
```

Comment:
This constant, `OS_VI_UNK28` is the one thing I wasn't sure how best to handle.
It's weird that the other `OS_VI_UNKxxx` are all hex flags; whereas here `28` is a decimal number not 0x28.